content/03.methods.md

```diff
@@ -27,10 +27,39 @@ We initially applied the beta-mixture quantile normalization (BMIQ) method [@doi
 We report uncorrected results in the main paper for simplicity.
 
 To make a fair comparison in each of the experiments displayed in the results, we used the intersection of TCGA samples having measurements for all of the datasets being compared in that experiment.
-This resulted in 3 distinct sets of samples: 9,074 samples shared between expression and mutation data, 7,981 samples shared between expression/mutation/27K methylation/450K methylation, and 5,282 samples shared between expression/mutation/27K methylation/450K methylation/RPPA/mutational signatures.
+This resulted in 3 distinct sets of samples: 9,074 samples shared between {expression, mutation} data, 7,981 samples shared between {expression, mutation, 27K methylation, 450K methylation}, and 5,282 samples shared between {expression, mutation, 27K methylation, 450K methylation, RPPA, mutational signatures}.
 When we dropped samples between experiments as progressively more data types were added, we observed that the dropped samples had approximately the same cancer type proportions as the dataset as a whole.
 In other words, samples that were profiled for one data type but not another did not tend to come exclusively from one or a few cancer types.
 Exceptions included acute myeloid leukemia (LAML) which had no samples profiled in the RPPA data, and ovarian cancer (OV) which had only 8 samples with 450K methylation data.
 More detailed information on cancer type proportions profiled for each data type is provided in (the supplement).
 
-(Include Venn diagrams of samples in supplement)
+### Training classifiers to detect cancer mutations
+
+We trained logistic regression classifiers to predict whether or not a given sample has a mutational event in a given target gene, using data from various -omics datasets as explanatory variables.
+As target genes, we considered 124 genes from the cancer-related gene dataset described in Vogelstein et al. 2013 [@doi:10.1126/science.1235122].
+For each target gene, in order to ensure that the training dataset was reasonably balanced (i.e. that there would be enough mutated samples to train a classifier), we included only cancer types with at least 15 mutated samples and at least 5% mutated samples.
+After filtering for sufficient mutated samples, 17 of the genes from the Vogelstein et al. gene set had no valid cancer types, leaving 107 genes with one or more valid cancer types to use in further analyses.
+
+Since our -omics datasets tend to have many dimensions and comparatively few samples, we applied an elastic net penalty to prevent overfitting [@doi:10.1111/j.1467-9868.2005.00503.x], as implemented in `scikit-learn`'s `SGDClassifier`.
+This approach is described more specifically in the methods section of Way et al. 2018 [@doi:10.1016/j.celrep.2018.03.046]; our classifiers follow the same general structure.
```

Comment:
Asking a reviewer to read a different paper's methods section is a good way to annoy a reviewer. You should reference that paper but also summarize it in brief here.

Reply:
Good to know - still learning to find the happy medium between too much methods section detail and not enough detail. I moved the citation and added a summary in f52a021; let me know if that's closer to what you had in mind.